prebid-server/hosting/bidder-specific-guidelines.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page_v2
+title: Bidder-Specific Guidelines for PBS Host Companies
+description: Bidder-Specific Guidelines for PBS Host Companies
+pid: 0
+sidebarType: 5
+---
+
+# Bidder-Specific Guidelines for PBS Host Companies
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Setting up a Bid Adapter
+
+In general, a PBS host company may want to run only the server-side bid adapters they require.
+
+PBS-Go enables most bidders by default, while PBS-Java doesn't
+enable any bidders out of the box.
```

Comment:
Nitpick: Can we remove the "while" and use two sentences? IMHO that comes across more neutral:

PBS-Go enables most bidders by default. PBS-Java doesn't enable any bidders by default.